src/holochainClient.js

```diff
@@ -34,7 +34,7 @@ async function initAndGetHolochainClient () {
   if (holochainClient) return holochainClient
   try {
     holochainClient = await hcWebClientConnect({
-      url: process.env.REACT_APP_DNA_INTERFACE_URL,
+      url: 'ws://localhost:3400/',
```

Comment:
why replace with hardcoded version?

Reply:
Can we implement this in this way:
```
const url = process.env.NODE_ENV === 'production' ? undefined : process.env.REACT_APP_DNA_INTERFACE_URL
```

Reply:
Same thought/reason as above.  I meant to clean up this hard-coded vars, so thanks for that catch!  @zo-el - I like the logical statement you proposed, and my intention is to implement this or something similar to leverage the benefit of a env var for testing, but following the 'hc-web-client' expected behavior for production environments for best coordination with the conductor and other hc apps.

Reply:
this just seems redundant

`const url = process.env.REACT_APP_DNA_INTERFACE_URL`

is the same as your suggestion @zo-el. This is what env variables are for. Set the variable how you need it in the specific environment.